tests/Fixer/FunctionNotation/PhpdocToReturnTypeFixerTest.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ *
+ * @requires PHP 7.0
```

Comment:
Shouldn't the tests run on PHP 5.6 as well and assert that nothing is changed?

Reply:
I've done like [VoidReturnFixerTest](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.11/tests/Fixer/FunctionNotation/VoidReturnFixerTest.php#L22) running the tests against 5.6 would only hit the `isCandidate` method:

https://github.com/Slamdunk/PHP-CS-Fixer/blob/58f511b4a1a56aa27379d31bdc303029fca5ae50/src/Fixer/FunctionNotation/PhpdocToReturnTypeFixer.php#L78-L81

Reply:
IMO `VoidReturnFixerTest` is not a good example. Current tests do not prove that the fixer does nothing on PHP 5.6. I would still remove the `@requires` annotation and either assert nothing is changed in any test case when running on PHP 5.6 or add specific tests for `isCandidate()`.

Reply:
Test onto PHP < 7.0 added.